src/sentry/replays/endpoints/organization_replay_index.py

```diff
@@ -21,6 +23,10 @@ def get(self, request: Request, organization: Organization) -> Response:
         except NoProjects:
             return Response({"data": []}, status=200)
 
+        serializer = ReplaySerializer(data=request.GET)
+        if not serializer.is_valid():
+            raise ParseError(serializer.errors)
+
         for key, value in request.query_params.items():
```

Comment:
instead of request.query_params you could try using `serializer` you initialize above -- don't think it makes a big difference since we're not doing any transforms atm.